swagger/v1/swagger.yaml

```diff
@@ -4,6 +4,176 @@ info:
+                      description:
+                        type: string
+                    required:
+                    - id
+                    - name
+                    - photo
+                    - purchase_price
+                    - rental_price
+                    - description
+                required:
+                - status
+                - message
+                - data
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                name:
+                  type: string
+                photo:
+                  type: string
+                purchase_price:
+                  type: number
+                rental_price:
+                  type: number
+                description:
+                  type: string
+              required:
+              - name
+              - photo
+              - purchase_price
+              - rental_price
+              - description
+    delete:
+      summary: Delete a specific console
+      tags:
+      - Console
+      parameters:
+      - name: id
+        in: path
+        required: true
+        schema:
+          type: integer
+      responses:
+        '200':
+          description: Console deleted successfully
+          content:
+            application/json:
+              examples:
+                example_0:
+                  value:
+                    status: Success
+                    message: Console deleted successfully
+                    data:
+                      id: 1
+                      name: NSWICTH
+                      photo: image_url
+                      purchase_price: 10000
+                      rental_price: 100
+                      description: Learn about Nintendo Switch Lite. It is compact, lightweight, and dedicated to handheld play. Making it perfect for gamers on the move.
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                  message:
+                    type: string
+                  data:
+                    type: object
+                    properties:
+                      id:
+                        type: integer
+                      name:
+                        type: string
+                      photo:
+                        type: string
+                      purchase_price:
+                        type: number
+                      rental_price:
+                        type: number
+                      description:
+                        type: string
+                    required:
+                    - id
+                    - name
+                    - photo
+                    - purchase_price
+                    - rental_price
+                    - description
+                required:
+                - status
+                - message
+                - data
```

Comment:
Please add tests for creating and deleting a post.

Here is the breakdown of the issue:

After pulling your changes this is what I see
![Screenshot from 2023-08-18 03-21-51](https://github.com/stephenkati/gaming_console_reservation_back_end/assets/47294794/e90c544b-e19f-4ab8-937f-7872c2ab46c8)

When I run `rake rswag:specs:swaggerize`, the documentation changed to:
![Screenshot from 2023-08-18 03-21-09](https://github.com/stephenkati/gaming_console_reservation_back_end/assets/47294794/cee3794c-97be-4c97-9bfa-a7bdfe3c3b6c)

Please add integration tests for the `post` and `delete`